### PR TITLE
Added PHP 7.4 to Travis build

### PR DESCRIPTION
**Pull request by lenaorobei**
## Overview
Magento Adobe Stock Integration needs to support PHP 7.4 for the next release.
See related issue https://github.com/magento/adobe-stock-integration/issues/1182 

## Technical Changes
- added PHP 7.4 to Travis build
- dropped PHP 7.1 support (EOL - https://www.php.net/supported-versions.php)
- upgraded PHPUnit version 
- fixed unit tests

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: php
 php:
- - 7.1
```
Comment (sivaschenko) [inline .travis.yml:3]: I don't think 7.1 check should be dropped as the SDK should still support this version for backward compatibility

File: composer.json [hunk 2]
```diff
@@ -9,12 +9,12 @@
         }
     ],
     "require": {
-        "php": "~7.1",
+        "php": "~7.2",
```
Comment (sivaschenko) [inline composer.json:12]: I don't think 7.1 support should be dropped as the SDK should still support this version for backward compatibility
```suggestion
        "php": "~7.1",
```

File: composer.json [hunk 3]
```diff
@@ -9,12 +9,12 @@
...
         "guzzlehttp/guzzle": "~6.0"
     },
     "require-dev": {
         "squizlabs/php_codesniffer": "~2.8",
-        "phpunit/phpunit": "~6.0",
+        "phpunit/phpunit": "^8.0",
```
Comment (sivaschenko) [inline composer.json:17]: Can we support both phpunit versions?
```suggestion
        "phpunit/phpunit": "~6.0||^8.0",
```

File: .travis.yml (hunk 1 above)
Comment (lenaorobei) [reply .travis.yml:3]: Please see previous Travis build with PHP 7.1.

File: composer.json (hunk 2 above)
Comment (lenaorobei) [reply composer.json]: Reverted change.

**Pull request merged**